internal/cri/server/cni_conf_syncer.go

```diff
@@ -97,13 +98,14 @@ func (syncer *cniNetConfSyncer) syncLoop() error {
 			}
 			log.L.Debugf("receiving change event from cni conf dir: %s", event)
 
-			lerr := syncer.netPlugin.Load(syncer.loadOpts...)
-			if lerr != nil {
-				log.L.WithError(lerr).
-					Errorf("failed to reload cni configuration after receiving fs change event(%s)", event)
+			if isValidFile(event.Name) {
+				lerr := syncer.netPlugin.Load(syncer.loadOpts...)
+				if lerr != nil {
+					log.L.WithError(lerr).
+						Errorf("failed to reload cni configuration after receiving fs change event(%s)", event)
+				}
+				syncer.updateLastStatus(lerr)
 			}
```

Comment:
you can add an
```suggestion
			} else {
			   log.L.Debugf("not recognized file extension: %s", event)
			}
```

Reply:
This is a great suggestion to assist with troubleshooting

Reply:
yeah, lot of scars 😄